vulnerabilities/azure-subdomain-takeover.yaml

```diff
@@ -0,0 +1,23 @@
+title: subdomain takeover using Azure
+slug: azure-subdomain-takeover
+cves: null
+affectedPlatforms:
+- Azure
+affectedServices: Traffic Manager
+image: null
+severity: Low
+discoveredBy:
+  name: Patrick Hudak
+  org: null
+  domain: https://0xpatrik.com/
+  twitter: https://twitter.com/0xpatrik
+disclosedAt: 10/08/2018
+exploitabilityPeriod: null
+knownITWExploitation: false
+summary: |
+  Patrick Hudak demonstrates possible subdomain takeover using the Traffic Manager in Azure.
+manualRemediation: |
+  None required
```

Comment:
This seems to still be possible - did MS do anything to change this functionality or make it harder to perform this type of attack against Traffic Manager subdomains? If not, we should use this section to describe how customers should protect themselves.